playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -100,8 +100,8 @@
   hosts: oo_masters_to_config:oo_nodes_to_config
   vars:
     target_version: "{{ '1.2' if deployment_type == 'origin' else '3.1.1.900' }}"
-    openshift_docker_hosted_registry_insecure: True
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

Comment:
> I think we could remove all openshift_docker_hosted_registry_insecure settings from playbooks and let openshift_docker_facts default the variable to True OR we could default to True wherever the value is accessed if the intent was to not persist this fact when the user didn't provide it.

Or we could default it to false in openshift_facts. I believe we need to make sure that this setting doesn't get exposed directly in the docker role, since it is an OpenShift specific configuration item, the hosted registry belongs to OpenShift (openshift_docker) and not Docker.